js.js

```diff
@@ -0,0 +1,18 @@
+const body = document.querySelector('body');
+const span = document.querySelector('span');
+const storage = window.sessionStorage;
+
+body.addEventListener('click', clicked);
+
+let counter = storage.getItem('counter') || 0;
+addCounter(span);
+
+function clicked() {
+  counter ++;
```

Comment:
let's move calculations to `addCounter`?

Reply:
this is not done?

Reply:
I'd say all of the logic from updateCounter can be moved here. It's too simple.